### PR TITLE
Fixed startup errors, removed/replaced deprecated packages, fixed test t...

### PR DESCRIPTION
**Pull request by jericsmith**
...o all run green, only cleaned up usings in files I touched

### CODE REVIEW

File: src/WebApiBook.IssueTrackerApi/app_start/CollectionJsonConfig.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
+namespace WebApiBook.IssueTrackerApi.SampleApi
```
Comment (glennblock) [inline src/WebApiBook.IssueTrackerApi/app_start/CollectionJsonConfig.cs:1]: What is this file for? How does that relate to the PR?

File: test/WebApiBook.IssueTrackerApi.AcceptanceTests/Features/CreatingIssues.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ namespace WebApiBook.IssueTrackerApp.AcceptanceTests.Features
 {
     public class CreatingIssues : IssuesFeature
     {
-        private Uri _issues = new Uri("http://localhost/issue");
+        private Uri _issues = new Uri("http://localhost:8080/issue");
```
Comment (glennblock) [inline test/WebApiBook.IssueTrackerApi.AcceptanceTests/Features/CreatingIssues.cs]: Port doesn't matter here as we're not sending anything across the network / everything is in memory. Why do you think the port needs to be there?

File: test/WebApiBook.IssueTrackerApi.AcceptanceTests/app_start/CollectionJsonConfig.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
+namespace WebApiBook.IssueTrackerApp.AcceptanceTests.SampleApi
```
Comment (glennblock) [inline test/WebApiBook.IssueTrackerApi.AcceptanceTests/app_start/CollectionJsonConfig.cs:1]: Why is this here?

File: src/WebApiBook.IssueTrackerApi/WebApiConfiguration.cs [hunk 4]
```diff
@@ -29,7 +29,7 @@ public static void Configure(HttpConfiguration config, IIssueStore issueStore =
        
         private static void ConfigureFormatters(HttpConfiguration config)
         {
-            config.Formatters.Add(new CollectionJsonFormatter());
+            config.Formatters.Add(new CollectionJsonFormatter.CollectionJsonMediaTypeFormatter());
```
Comment (glennblock) [inline src/WebApiBook.IssueTrackerApi/WebApiConfiguration.cs:32]: This is really just a matter of preference. I prefer having the using rather than fully qualifying the references.

**Comment by glennblock** [comment]
Hi @jericsmith 

Thanks for the effort. I sent you a bunch of questions / comments.

File: test/WebApiBook.IssueTrackerApi.AcceptanceTests/Features/CreatingIssues.cs (hunk 2 above)
Comment (jericsmith) [inline test/WebApiBook.IssueTrackerApi.AcceptanceTests/Features/CreatingIssues.cs]: If I remember correctly there was a test condition dependent upon it.   I'll take a look asap

--- Original Message ---

**Comment by jericsmith** [comment]
Ok.   I'll revisit the code asap to answer.  Thanks

--- Original Message ---

File: src/WebApiBook.IssueTrackerApi/WebApiConfiguration.cs (hunk 4 above)
Comment (jericsmith) [inline src/WebApiBook.IssueTrackerApi/WebApiConfiguration.cs:32]: with the CollectionJsonFormatter package update, CollectionJsonFormatter is a namespace. The test was failing.

File: test/WebApiBook.IssueTrackerApi.AcceptanceTests/app_start/CollectionJsonConfig.cs (hunk 3 above)
Comment (jericsmith) [inline test/WebApiBook.IssueTrackerApi.AcceptanceTests/app_start/CollectionJsonConfig.cs:1]: added with package updates

File: test/WebApiBook.IssueTrackerApi.AcceptanceTests/Features/CreatingIssues.cs (hunk 2 above)
Comment (jericsmith) [inline test/WebApiBook.IssueTrackerApi.AcceptanceTests/Features/CreatingIssues.cs]: my bad

**Comment by glennblock** [comment]
This is sending too many changes in a single PR. Can you break out each significant change into a sep PR so that each PR only addresses one thing?

Thanks!
